egui_extras/src/table.rs

```diff
@@ -412,6 +457,16 @@ impl<'a> TableBody<'a> {
 
         self.row_nr += 1;
     }
+
+    pub fn buffer(&mut self, height: f32) {
```

Comment:
needs a docstring or a better name (`advance_height`?)

Reply:
I added a docstring and removed the `pub`, but `buffer` still makes more sense to me semantically -- but since you're the maintainer if you have a strong preference I'll go with whatever your suggestion is.

Reply:
buffer is so many different things (including a verb). `add_buffer` or `add_space`?